smart-contract/README.md

```diff
@@ -306,7 +306,7 @@ Grant for bank 1
 
 ```bash
 provenanced tx marker grant-authz \
-    tp14hj2tavq8fpesdwxxcu44rty3hh90vhuz3ljwv \
+    tp14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9s96lrg8 \
```

Comment:
1.8 seems to have changed the length of the smart contract addresses on instantiation